baseframe/static/js/baseframe-bootstrap.js

```diff
@@ -292,6 +292,28 @@ window.Baseframe.Forms = {
         }
       }
     });
+  },
+  handleAjaxPost: function(config) {
```

Comment:
This isn't a generic AJAX POST handler, right? Seems like it's a form handler. So, `submitForm` perhaps?

Reply:
Yes, it is a generic ajax post handler. Not sure why it seemed like a form handler. 
'application/x-www-form-urlencoded; charset=UTF-8' is the default contentType in jquery ajax

Reply:
> Not sure why it seemed like a form handler.

Because line 297 explicitly looks for a HTML element (likely a `form` element) with a `submit` button and adds a `click` handler to the button.

Reply:
By not generic I meant to say that, I can't issue a plain JSON POST request without having a form element in my HTML.

Reply:
Fixed